.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,5 +1,8 @@
-#Tue Jan 24 17:02:29 PST 2012
 eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.6
```

Comment:
Setting compiler settings seems good. I've not used Eclipse for a fairly long time, isn't it possible to just use the `pom.xml` to replace most of the direct Eclipse configuration?